.github/release-drafter.yml

```
@@ -0,0 +1,19 @@
+prerelease: true
+categories:
+  - title: 'Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: 'Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'
+  - title: 'Maintenance'
+    label: 'chore'
```

Are we going to uses the Added/Removed/Changed/Fixed categories from:
https://keepachangelog.com/en/1.0.0/

Let's also use the labels from our GitHub.
I added a security and deprecation labels.

```suggestion
  - title: '### Added'
    labels:
      - 'C-enhancement'
      - 'C-design'
  - title: '### Changed'
    label: 'A-dependencies'
  - title: '### Deprecated'
    label: 'C-deprecation'
  - title: '### Removed'
    label: 'C-cleanup'
  - title: '### Fixed'
    label: 'C-bug'
  - title: '### Security'
    label: 'C-security'
```